lib/recurly/token.js

```diff
@@ -65,6 +66,8 @@ function token (options, done) {
   var data = normalize(fields, options, { parseCard: true });
   var inputs = data.values;
 
+  inputs.fraud = this.fraud.params(inputs);
+
```

Comment:
LEt's move this into the parent only block, since fraud data collection should only occur in that context.